src/api/api.ts

```diff
@@ -1 +1,7 @@
-export const BASE_URL = 'https://mate.academy/students-api';
+export const BASE_URL = 'https://mate.academy/students-api/';
```

Comment:
Do you really need a trailing slash? 
If you use it here so you don't need to use it below with any endpoints. But it will look so weird. So there prefer to use BASE_URL without trailing slash.